store/store.go

```diff
@@ -471,6 +506,10 @@ func calcBlockHashKey(hash []byte) []byte {
 	return []byte(fmt.Sprintf("BH:%x", hash))
 }
 
+func calcTxHashKey(hash []byte) []byte {
```

Comment:
Renaming this function would be helpful since it only converts the hash to a string, without performing any calculations. It could be given a more accurate name, or alternatively, the operation could be carried out inline instead of using a separate function for it.

Reply:
I'm following a general pattern of generating keys in blockstore. view https://github.com/celestiaorg/celestia-core/blob/1d70a8e659d6dc1dee5057ba47bd0d92f99d0e59/store/store.go#L454

Reply:
Oh, I see. Then, that's an unusual naming convention in my opinion. Though, nothing to be done about it in this PR.